source/material_model/reactive_fluid_transport.cc

```diff
@@ -274,7 +285,6 @@ namespace aspect
     }
 
 
-
```

Comment:
This line actually needs to stay (there are always three empty lines between functions in the cc file).